R/assertions.R

```diff
@@ -1386,15 +1390,15 @@ assert_expr_list <- function(arg, # nolint
   }
 
   if (!inherits(arg, "list")) {
-    err_msg <- sprintf(
-      paste0(
-        "`%s` must be a named list of expressions but it is %s\n",
-        "\u2139 To create a list of expressions use `exprs()`"
-      ),
-      arg_name(substitute(arg)),
-      what_is_it(arg)
+    cli_abort(
+      message = message %||%
+        c("Argument {.arg {arg_name}} must be a named list of expressions
```

Comment:
```suggestion
        c("Argument {.arg {arg_name}} must be a list of expressions
```

should we remove "named"?

Reply:
Yes